src/main/java/seedu/tasklist/logic/commands/ClearCommand.java

```diff
@@ -16,6 +16,9 @@ public ClearCommand() {}
     @Override
     public CommandResult execute() {
         assert model != null;
+      	Task.INCOMPLETE_COUNTER=0;
```

Comment:
oh, here task is the class Task itself not an instance of Task and hence it's this way.